xcode/Subconscious/Shared/Components/Deck/DeckView.swift

```diff
@@ -119,6 +119,8 @@ enum DeckAction: Hashable {
     case succeedMergeEntry(parent: Slashlink, child: Slashlink)
     case requestUpdateAudience(_ address: Slashlink, _ audience: Audience)
     case succeedUpdateAudience(_ receipt: MoveReceipt)
+    case requestAssignNoteColor(_ address: Slashlink, _ color: NoteColor)
+    case succeedAssignNoteColor(_ address: Slashlink, _ color: NoteColor)
```

Comment:
I think I would recommend just passing Color at the level of actions. NoteColor can act as a constant we use when assigning colors.